src/Bootstrap/less/theme/page-display-package-v2.less

```
@@ -185,6 +185,47 @@
         margin-top: 3px;
       }
     }
+
+    .sidebar-headers {
+      font-size: 16px;
+      font-weight: 600;
+      color: #666666;
+      margin-bottom: 12px;
+    }
+
+    .sidebar-links {
+      font-size: 16px; 
+    }
+
+    #title-links {
+      font-size: 14px;
+      font-weight: 400;
+    }
+
+    .sidebar-section {
+      margin-bottom: 32px;
+    }
+
+    .sidebar-table {
+      margin-top: 8px;
+
+      tr {
+        border-style: hidden;
+      }
+
+      th {
+        text-align: center;
+        font-size: 14px;
+        font-weight: 400;
+        color: #666666;
```

This can also be `@gray-light`:

```suggestion
        color: @gray-light;
```

---

done 👍